src/main/java/com/cleanroommc/groovyscript/compat/mods/tinkersconstruct/Alloying.java

```diff
@@ -89,11 +99,13 @@ public boolean removeByInputsAndOutput(FluidStack output, FluidStack... inputs)
         return false;
     }
 
+    @MethodDescription(description = "groovyscript.wiki.removeAll")
     public void removeAll() {
         TinkerRegistryAccessor.getAlloyRegistry().forEach(this::addBackup);
         TinkerRegistryAccessor.getAlloyRegistry().forEach(TinkerRegistryAccessor.getAlloyRegistry()::remove);
     }
 
+    @MethodDescription(description = "groovyscript.wiki.streamRecipes")
```

Comment:
should indicate the method type is `QUERY` to document properly
```suggestion
    @MethodDescription(description = "groovyscript.wiki.streamRecipes", type = MethodDescription.Type.QUERY)
```